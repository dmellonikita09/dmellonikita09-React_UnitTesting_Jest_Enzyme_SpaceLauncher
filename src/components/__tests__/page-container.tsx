import React from 'react';
import { mount } from 'enzyme';
import PageContainer from '../page-container';

describe('Page Container', () => {
  it('should contain passed page content', () => {
    const mockPageOne = (<div id='page1'/>), mockPageTwo = (<div id='page2'/>)
    const wrapper = mount(
      <PageContainer>
        {mockPageOne}
        {mockPageTwo}
      </PageContainer>);
    expect(wrapper).toMatchSnapshot();
  });
});
