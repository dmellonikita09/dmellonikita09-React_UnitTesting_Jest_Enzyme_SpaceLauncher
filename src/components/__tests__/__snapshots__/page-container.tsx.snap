// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Container should contain passed page content 1`] = `
<PageContainer>
  <Styled(div)>
    <div
      className="css-k2u1cc"
    />
  </Styled(div)>
  <Styled(div)>
    <div
      className="css-1on771l"
    >
      <div
        id="page1"
      />
      <div
        id="page2"
      />
    </div>
  </Styled(div)>
</PageContainer>
`;
